Source/Web/app/Login.html

```diff
@@ -28,28 +28,32 @@
               <a href="Register.html">Sign Up</a>
             </ul>
         </div>
-        
-        <div class="container">            
+        <br />
+	<br />
+	<div class="center">
+        <div class="container">  
+	
             <div class="row">
-                <div class="col-md-2 col-md-offset-5">
+                <div class="col-md-3  ">
                     <div class="loginBlock" align="center">                    
                     
 <!--                     We remove the form because it reloads the page -->
 <!--                     Which we don't want. -->
 <!--                     <form action="" method="POST" > -->
-                        <h1>Log in to Harvest</h1>
+                        <h1><b>Log in to Harvest </b></h1>
```

Comment:
Why is this bold, when it's not on the register page.

Reply:
Just forgot to make the other bold. Thanks for pointing it out.